org-formation/organization.yaml

```diff
@@ -285,3 +286,15 @@ Organization:
         Project: iAtlas
         budget-alarm-threshold: 2000
         budget-alarm-threshold-email-recipient: aws-iatlas-prod@sagebase.org
+
+  WorkflowsNextflowDevAccount:
```

Comment:
this portion was already done in PR #131 ?

i think you may need to bring this current with master.